website/docs/configuration/variables.html.md

```diff
@@ -220,6 +208,91 @@ that includes the sentences given in `error_message`. The error message string
 should be at least one full sentence explaining the constraint that failed,
 using a sentence structure similar to the above examples.
 
+### Sensitive
+
+[inpage-sensitive]: #sensitive
+
+-> This feature was introduced in Terraform CLI v0.14.0.
+
+The `sensitive` argument on a variable block is a boolean value that, when provided, limits the output of the Terraform `plan` or `apply` when that variable is used. A provider can define [an attribute as sensitive](/docs/extend/best-practices/sensitive-state.html#using-the-sensitive-flag), which prevents the value of that attribute from being displayed in logs or regular output. The `sensitive` argument on variables allows users to replicate this behavior for values in their configuration, by defining a variable as `sensitive`.
```

Comment:
I might rephrase this sentence a bit, inspired by our sensitive outputs documentation (https://www.terraform.io/docs/configuration/outputs.html):

Setting a variable as `sensitive` argument prevents Terraform from showing its value in the `plan` and `apply` output.